assisted-events-scrape/workers/cluster_events_worker.py

```diff
@@ -174,9 +176,13 @@ def _cluster_checksum(self, doc: dict) -> dict:
             doc_copy["hosts"].sort(key=by_id)
         return get_dict_hash(doc_copy)
 
+    def _enrich_cluster(self, doc: dict):
+        doc["cluster_state_id"] = str(uuid5(CLUSTER_STATE_ID_NAMESPACE, self._cluster_checksum(doc)))
```

Comment:
@falox This is where we add the `cluster_state_id` field

Reply:
Is this unique even if the id, updated_at are the same? And what happens if, for any reason, there are 2 rows with the same fields: wIll the `cluster_state_id` be the same?

Reply:
This ID is calculated from the checksum of the document (it is basically the checksum excluding some frequently changing fields turned into UUID format). If by chance there are multiple states the same fields, it won't get updated (it uses the checksum as unique ID of the document, therefore we will never have 2 documents with the same field value bar undesired and unlikely collisions)